css/properties/column-width.json

```diff
@@ -97,6 +97,38 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "auto": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1.5"
+              },
+              "firefox_android": "mirror",
```

Comment:
This doesn't look right: Even if we ignore the prefixes, which sounds reasonable for the sub-features, then Firefox Android only supports the parent feature from version 4, doesn't it?

Reply:
The first release of Firefox Android was version 4, so mirroring is accurate -- the mirroring script recognizes this and automatically adjusts the version number accordingly!

Reply:
Ah, makes sense, thanks for clarifying!